File: src/main/java/ru/citeck/idea/codeinsight/forms/components/Panel.java
package ru.citeck.idea.codeinsight.forms.components;

import ecos.com.fasterxml.jackson210.annotation.JsonProperty;

public class Panel extends Container {

    public static final String TYPE = "panel";

    @Override
    public String getType() {
        return TYPE;
    }

    @Override
    @JsonProperty("title")
    public String getLabel() {
        return super.getLabel();
    }
}

File: src/main/kotlin/ru/citeck/idea/artifacts/action/codeinsight/forms/components/Panel.kt
package ru.citeck.idea.artifacts.action.codeinsight.forms.components

import ecos.com.fasterxml.jackson210.annotation.JsonProperty

class Panel : Container() {

    companion object {
        const val TYPE: String = "panel"
    }

    @JsonProperty("title")
    override fun setLabel(label: String) {
        super.setLabel(label)
    }

    @JsonProperty("title")
    override fun getLabel(): String {
        return super.getLabel()
    }

    override fun getType(): String {
        return TYPE
    }
}
